tests/publiccloud/ipa.pm

```diff
@@ -152,15 +152,15 @@ sub az_run_ipa {
 
     my $credentials = <<EOT;
 {
-  "clientId": "$clientid", 
-  "clientSecret": "$clientsecret", 
-  "subscriptionId": "$subscription", 
-  "tenantId": "$tenantid", 
-  "activeDirectoryEndpointUrl": "https://login.microsoftonline.com", 
-  "resourceManagerEndpointUrl": "https://management.azure.com/", 
-  "activeDirectoryGraphResourceId": "https://graph.windows.net/", 
-  "sqlManagementEndpointUrl": "https://management.core.windows.net:8443/", 
-  "galleryEndpointUrl": "https://gallery.azure.com/", 
+  "clientId": "$clientid",
```

Comment:
I think my editor removed the trailing spaces automatically, but better like this IMO.

Reply:
sure but can you please cherry-pick this into separate commit ?

Reply:
Sorry my fault, I updated my vimrc to avoid it :)

Reply:
I will actually leave it as it was, since this PR has nothing to do with trailing whitespaces :)